webapp/channels/src/reducers/views/drafts.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {combineReducers} from 'redux';
+import {GenericAction} from 'mattermost-redux/types/actions';
+import {ActionTypes} from 'utils/constants';
+
+function remotes(state: Record<string, boolean> = {}, action: GenericAction) {
+    switch (action.type) {
+    case ActionTypes.SET_DRAFT_SOURCE:
+        return {
+            ...state,
+            [action.data.key]: action.data.isRemote,
+        };
+    default:
+        return state;
+    }
+}
+
+export default combineReducers({
+    remotes,
```

Comment:
Mind adding a short comment here to describe what this field is? We haven't done that for most of the view reducers, but I could see someone not understanding what this is used for when reading the code